### DISCUSSION TITLE
How to use the wildcard in input.json

### DISCUSSION THREAD

**Discussion by goodluck-xyz** [deepmodeling/deepmd-kit#1911] [Q&A] [answered]
In deepmdkit, Using the wildcard in input.json as follow,
![image](https://user-images.githubusercontent.com/54764135/189787516-de495fad-5580-4464-b41b-486db0d8a473.png)
An error occupys as follow
![image](https://user-images.githubusercontent.com/54764135/189787629-33d91f14-4a8b-4c9e-a87b-b79159482fd7.png)
However, in dpgen,  the similar wildcard is ok, like this
![image](https://user-images.githubusercontent.com/54764135/189788181-c24387ba-1ba2-415e-b9d2-02f06b25e851.png)
What is the difference in the use of wildcards between dpgen and deepmdkit?

**Comment by njzjz** [comment, marked as answer]
DeePMD-kit does not support wildcards.

**Reply by goodluck-xyz** [reply]
Thank you.
